Bottom: a168f8a03748927cd70d01bafff8e465be163933
Top:    eaa540c68dddb8a5a16998b053c506b801230db5
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-06 16:34:47 +0000

Refresh of play-with-quad-edge-data-STEP

---

diff --git a/src/Makefile.am b/src/Makefile.am
index 35d6178..60807b7 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -169,6 +169,7 @@ PCB_SRCS = \
 	hid/common/object3d.h \
 	hid/common/quad.c \
 	hid/common/quad.h \
+	hid/common/step_id.h \
 	hid/common/vertex3d.c \
 	hid/common/vertex3d.h \
 	hid/hidint.h 
diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index 263c089..cfc0861 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -5,6 +5,7 @@
 
 #include <glib.h>
 
+#include "step_id.h"
 #include "quad.h"
 #include "vertex3d.h"
 #include "contour3d.h"
diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index 015d575..6e8c5d3 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -15,6 +15,7 @@
 #include "gui-pinout-preview.h"
 #include "pcb-printf.h"
 
+#include "hid/common/step_id.h"
 #include "hid/common/quad.h"
 #include "hid/common/vertex3d.h"
 #include "hid/common/contour3d.h"
diff --git a/src/hid/step/object3d_step.c b/src/hid/step/object3d_step.c
index b781307..fb0b142 100644
--- a/src/hid/step/object3d_step.c
+++ b/src/hid/step/object3d_step.c
@@ -4,6 +4,7 @@
 
 #include <glib.h>
 
+#include "hid/common/step_id.h"
 #include "hid/common/quad.h"
 #include "hid/common/vertex3d.h"
 #include "hid/common/contour3d.h"
